CityOfDetroit/bloom
# 1338/sro removal

## Issue
This PR addresses #1338.

## Description

In this PR, I removed the SRO option from public filtering, creating units on the partners side, and all other mentions of the option. Additionally, I included a migration script that removes the column from the unit_types table which was necessary since the filter form options are pulled from the backend listings meta data. As of right now, there doesn't seem to be any public listings that have SROs on the live Detroit site, but I added a line in the migration script to change any existing SROs to be classified as Studio on the off chance that there's one in the drafts or before this is merged in. Just want to make sure that this SRO -> Studio decision still stands @kathyccheng ?

## How Can This Be Tested/Reviewed?

This can be tested by going to the Detroit public site, going to listings, and filtering. You should notice that the SRO option is no longer available. Then on the Partner's side, click add a listing, scroll down and click to create a unit group, and notice that  the SRO classification is not available there either.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
